src/html/layout.html

```diff
@@ -21,19 +21,21 @@
   <div role="navigation" aria-label="related navigation">
     <div class="sp-logo-container sp-not-mobileview sp-mobileview">
       <a class="sp-logo-link" data-auto-route="true" href="/">
-        <img height="40px" id="site-logo" class="logo-big" alt="Sponge Forums" src="{{pathto("_static/spongelogo.png", 1)}}">
+        <img height="40px" id="site-logo" class="logo-big" alt="" src="{{pathto("_static/spongie-mark.svg", 1)}}">
+        <span id="sp-site-title">Sponge</span>
       </a>
       <div class="sp-logo-bg"></div>
       <div class="sp-logo-chevron"><i class="fa fa-fw fa-chevron-down"></i></div>
       <div class="sp-logo-menu sp-skip-handler">
         <ul class="sp-logo-dropdown" id="ddleft">
           <a href="https://www.spongepowered.org"><li><i class="fa-fw fa fa-home"></i>Homepage</li></a>
-          <a href="https://forums.spongepowered.org"><li><i class="fa-fw fa fa-comment"></i>Forums</li></a>
+          <a href="https://forums.spongepowered.org"><li><i class="fa-fw fa fa-comments"></i>Forums</li></a>
           <a href="https://github.com/SpongePowered"><li><i class="fa-fw fa fa-code"></i>Code</li></a>
-          <a class="sp-forums-home" href="https://docs.spongepowered.org"><li class="active"><i class="fa-fw fa fa-book"></i>SpongeDocs</li></a>
-          <a href="https://jd.spongepowered.org"><li><i class="fa-fw fa fa-graduation-cap"></i>JavaDocs</li></a>
+          <a class="sp-forums-home" href="https://docs.spongepowered.org"><li class="active"><i class="fa-fw fa fa-book"></i>Docs</li></a>
+          <a href="https://jd.spongepowered.org"><li><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</li></a>
           <a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><li><i class="fa-fw fa fa-plug"></i>Plugins</li></a>
-          <a href="https://forums.spongepowered.org/t/sponge-downloads/11448"><li><i class="fa-fw fa fa-download"></i>Get Sponge</li></a>
+          <a href="https://www.spongepowered.org/#downloads"><li><i class="fa-fw fa fa-download"></i>Downloads</li></a>
```

Comment:
Should link to `https://www.spongepowered.org/downloads` (`/` instead of `#`), we have a dedicated downloads page now